recipe/meta.yaml

```diff
@@ -1,15 +1,20 @@
 {% set name = "lerc" %}
-{% set version = "3.0" %}
+{% set version = "4.0.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/Esri/lerc/archive/v{{ version }}.tar.gz
-  sha256: 8c0148f5c22d823eff7b2c999b0781f8095e49a7d3195f13c68c5541dd5740a1
+  # temporarily, version starts with js_*, remove it when a new version is available.
```

Comment:
There is a tag, but it is still marked as unreleased. We should stick to version 4.0.0

Reply:
Done